README.md

```diff
@@ -1 +1,116 @@
-# intro-to-tests-activity
+# Intro To Tests Activity 
+This repository is part of the class activity for Intro to Tests in Unit 1.
+
+## Learning Goals
+- Practice identifying test cases
+- Practice writing tests with pytest
+
+### Part 1: One-Time Setup
+
+Follow these directions once when you start working on this activity **during roundtables**:
+
+1. Navigate to your projects folder named `projects`
+
+```bash
+$ cd ~/Developer/projects
+```
+
+1. In Github click on the green "Code" button in Github and then copy the URL. This will allow you to download a copy of this project into your projects folder. This command makes a new folder called `intro-to-tests-activity`, and then puts the project into this new folder. 
```

Comment:
```suggestion
2. In Github click on the green "Code" button in Github and then copy the URL. This will allow you to download a copy of this project into your projects folder. This command makes a new folder called `intro-to-tests-activity`, and then puts the project into this new folder. 
```

Reply:
The Pt. 1 numbering looks like it got a little weird in the port, I made some suggestions, but they may not be exactly what the original had.